src/cryptonote_core/tx_pool.cpp

```diff
@@ -54,6 +54,9 @@ namespace cryptonote
 {
   namespace
   {
+    //FIXME: constants such as these should at least be in the header,
+    //       but probably somewhere more accessible to the rest of the
+    //       codebase.
```

Comment:
They're protocol-wide constants.  While they may only be used here right now, they're versioned settings and I think they should be kept with all other versioned settings if possible.  Probably could change from "FIXME" to "TODO" though.